ui/src/components/microsegmentation/AddSegmentation.js

```diff
@@ -604,6 +605,16 @@ class AddSegmentation extends React.Component {
         return error;
     }
 
+    // isScopeOnPrem returns true if onprem is a selected option
+    isScopeOnPrem() {
+        for (let i = 0; i < this.state.PESList.length; i++) {
+            if (this.state.PESList[i].scopeonprem == 'false') {
```

Comment:
Since initial state is:
```
                scopeonprem: 'true',
                scopeaws: 'false',
                scopeall: 'false',
```
In case the user checks `aws` in addition to on prem - it will return `true` instead of `false` - correct?
consider maybe we should change the method name to `isOnlyOnPremScope` ..